readme.md

```diff
@@ -24,74 +24,75 @@ This page is in constant WIP, please refer to it regularly.
 # The carwow Engineering Principles
 
 
-__Build what matters__ 
-  
+__Build what matters__
+
   * Engineering effort is a scarce commodity. It should only be applied to problems that “move the needle” for the company
   * The biggest achievement of an Engineer at carwow is to provide positive business impact with no line of code written (maybe even removing some)
   * We don't manage our own server, or have a self hosted continuous integration software.
-    
-    
+
+
 __Do *big* things in small steps__
-  
+
   * Breaking down big features in small steps doesn’t mean we work only on small incremental improvement. A longer term vision or objective help us achieve great things without sacrificing short term value.
-    
-    
+  * Feature-specific toggles are a good way to release early and often. By coupling each well-defined feature with its own toggle, we can better control when and where to activate it.
```

Comment:
@soulnafein you said https://github.com/carwow/engineering_principles/pull/4#discussion_r256436126

How about the following change: